src/Approximations/overapproximate.jl

```diff
@@ -1963,6 +1963,58 @@ function overapproximate(qm::QuadraticMap{N, <:AbstractZonotope}, ::Type{<:Zonot
     return Zonotope(d, remove_zero_columns(h))
 end
 
+"""
+    overapproximate(QM::QuadraticMap{N, <:SparsePolynomialZonotope},
+                    ::Type{SparsePolynomialZonotope}) where {N}
+
+Return a sparse polynomial zonotope overapproximating the quadratic map of a
+sparse polynomial zonotope.
+
+### Input
+
+- `QM` -- quadratic map of a sparse polynomial zonotope
+- `SparsePolynomialZonotope` -- target type
+
+### Output
+
+A sparse polynomial zonotope overapproximating the quadratic map of a sparse
+polynomial zonotope.
+
+### Algorithm
+
+This method implements Proposition 13 of [1].
+
+[1] N. Kochdumper and M. Althoff. Sparse Polynomial Zonotopes: A Novel Set Representation for
+Reachability Analysis. Transactions on Automatic Control, 2021.
+"""
+function overapproximate(QM::QuadraticMap{N, <:SparsePolynomialZonotope}, ::Type{SparsePolynomialZonotope}) where {N}
+    PZ = QM.X
+    Q = QM.Q
+
+    p = nparams(PZ)
+    q = ngens_indep(PZ)
+
+    c = center(PZ)
+    G = genmat_dep(PZ)
+    GI = genmat_indep(PZ)
+    Ĝ = hcat(G, GI)
+    Ê = cat(expmat(PZ), Matrix(1 * I, q, q); dims=(1, 2))
```

Comment:
At some point we have to review these constructions to make use of sparsity. But we can keep it for now.

Reply:
https://github.com/JuliaReach/LazySets.jl/issues/3068 opened an issue for it